app/views/orders/_order.html.haml

```diff
@@ -1,5 +1,8 @@
 %li
   = "Order for #{order.stamp}:"
   = render partial: 'orders/meal', collection: order.meals
-  = "| #{link_to "#{t('views.orders.edit')}", edit_menu_order_path(order.menu, order)}"
-  = "| #{link_to "#{t('views.orders.delete')}", menu_order_path(order.menu, order), method: :delete} "
+  - if (can? :manage, order.object) && allow_modification?(order.menu)
```

Comment:
Why not `if can?(:manage, order.object) && allow_modification?(order.menu)`? Seems more readable